### ISSUE TITLE
New generator with oil

### ISSUE THREAD

**Issue by Hexrotor** [Anuken/Mindustry-Suggestions#2782]
### Describe the content or mechanics you are proposing.

It's fuel cell that works with oil.
Name: Fuel Generator
Size: 2x2
Base Power Generation: 610 power units/second
Input: 6 Oil/sec

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

Oil is very important in the game, but it has too little use. Sometimes the oil field is nearby the core, maybe we can use it to generate power directly.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [x] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by Hexrotor** [comment]
Of course we can use coal centrifuge. The game has no generator that power ranging from 400 to 800, maybe this is a good transitional item.

**Comment by oinkymoo** [comment]
as pointed out in an old suggestion, oil is useless without titanium, this was a proposed solution that i support

**Comment by itcannotbe** [comment]
Actually oil was usable with the steam generator long time ago, but then came the coal extractor.......

Also why do we need in betweens again?

**Comment by Hexrotor** [comment]
> Actually oil was usable with the steam generator long time ago, but then came the coal extractor.......
> 
> Also why do we need in betweens again?

Maybe you are right. In my opinions, fuel cells are not steam driven, it can work without water so may cost fewer items to build.

**Comment by itcannotbe** [comment]
> > Actually oil was usable with the steam generator long time ago, but then came the coal extractor.......
> > Also why do we need in betweens again?
> 
> Maybe you are right. In my opinions, fuel cells are not steam driven, it can work without water so may cost fewer items to build.

Ah yes, bAlAnCiNg.......

**Comment by Absolute359** [comment]
Imagine how much coal you get out of 6oil/sec
Thats enough to power 3 steams wich 330pwr/sec
Thats 948 including -42 of the coal centri
More then 4.5 times more efficient then 210pwr out of 6oil

**Comment by Hexrotor** [comment]
> Imagine how much coal you get out of 6oil/sec
> Thats enough to power 3 steams wich is 330pwr/sec
> Thats 948 including -42 of the coal centri
> More then 4.5 times more efficient then 210pwr out of 6oil

That's true… but sometimes steam generator must work with water extractor, that means less power… Of course we can spend more items to build 3 or more steam generator to get more power, 

> Nvm. You said 610/sec wich sounds very OP for a 2x2

Maybe this is a bit unreasonable, Anuke will change it if he likes😂

**Comment by OnyxFinger** [comment]
I like it since having to refine oil into coal if you want power seems like an unnecessary step, even if it's unrealistic to some ?
I support this idea
